addon/components/frost-date-picker.js

```diff
@@ -50,7 +50,7 @@ export default Component.extend(EventsProxyMixin, {
   @readOnly
   @computed('value')
   _value (value) {
-    if (value === undefined) {
+    if (!value) {
```

Comment:
I like that this covers both `undefined` and `''`, but there are other falsey values (`0`) which probably we still want to mark as invalid.  I like this, but would be happier if it were more explicitly only for values that can represent emptiness (`undefined`, `''`).

Reply:
Hmmmmmmm, sure, I'll go with it

Reply:
Thank you!